olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/CatalogVersionRequest.h

```diff
@@ -33,46 +33,54 @@ namespace dataservice {
 namespace read {
 
 /**
- * @brief The CatalogVersionRequest class encapsulates the fields required to
- * request the Catalog configuration.
+ * @brief Encapsulates the fields required to request catalog configuration.
  */
 class DATASERVICE_READ_API CatalogVersionRequest final {
  public:
   /**
-   * @brief Mandatory for versioned layers; the beginning of the range of
-   * versions you want to get (exclusive). By convention -1 indicates the
-   * initial version before the first publication. After the first publication,
-   * the catalog version is 0.
-   * @return the start version.
+   * @brief Gets a catalog start version (exclusive) for the request.
+   *
+   * Mandatory for versioned layers.
+   * By convention -1 indicates the initial version before the first
+   * publication. After the first publication, the catalog version is 0.
+   *
+   * @return The catalog start version.
    */
   inline int64_t GetStartVersion() const { return start_version_; }
 
   /**
-   * @brief WithStartVersion sets the start version. See ::GetStartVersion() for
-   * usage.
-   * @param startVersion an int64_t
-   * @return a reference to the updated CatalogVersionRequest
+   * @brief Sets the catalog start version.
+   *
+   * @see `::GetStartVersion()` for usage.
+   * @param startVersion The catalog start version.
+   * @return A reference to the updated `CatalogVersionRequest` instance.
    */
   inline CatalogVersionRequest& WithStartVersion(int64_t startVersion) {
     start_version_ = startVersion;
     return *this;
   }
 
   /**
-   * @brief BillingTag is an optional free-form tag which is used for
-   * grouping billing records together. If supplied, it must be between 4 - 16
-   * characters, contain only alpha/numeric ASCII characters  [A-Za-z0-9].
-   * @return the billing tag, or boost::none if not set.
+   * @brief Gets a billing tag to group billing records together.
+   *
+   * The billing tag is an optional free-form tag that is used for grouping
+   * billing records together. If supplied, it must be 4&ndash;16 characters
```

Comment:
```suggestion
   * billing records together. If supplied, it must be 4-16 characters
```
Would a real dash, i.e. `-`, be cleaner? The encoding is nice for HTML but it's harsh on humans:)

Reply:
Agree.
Added the en dash sign (–) instead of `&ndash;`.